src/http/Response.cpp

```diff
@@ -0,0 +1,208 @@
+#include "Response.hpp"
+
+namespace http
+{
+
+Response::Response(void) {}
+
+Response::~Response(void) {}
+
+void
+Response::load_http_request(Request &req)
```

Comment:
Do we rename this variable ?
```suggestion
Response::load_http_request(Request &request)
```

Reply:
ok